__tests__/integration/fixtures/bug-include-path-edit-uri/api/mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: "test"
+site_description: "This is an api."
+repo_url: https://github.com/backstage/mkdocs-monorepo-plugin
```

Comment:
It would be good to have another fixture that covers the case when neither `repo_url` nor `edit_uri` are set in the child `mkdocs.yml`.

I believe your code already handles this case?  Just want to make sure it's tested.